nano/secure/account_info.hpp

```diff
@@ -29,4 +29,24 @@ class account_info final
 	uint64_t block_count{ 0 };
 	nano::epoch epoch_m{ nano::epoch::epoch_0 };
 };
+
+/**
+ * Account info as of DB version 22.
+ * This class protects the DB upgrades from future changes of the account_info class.
```

Comment:
It wasn't immediately clear to me the purpose of this class or how it protected DB upgrades.

Maybe "This is a snapshot of the account_info table at v22 which needs to be read for the v22 to v23 upgrade.

Reply:
Changed the comment